[Kiali][master] update kiali version

jmazzitelli opened the pull request:
This upgrades Kiali to v0.20. This is built on the master branch. We should cherry pick this over to the release-1.2 branch - it should be a simple cherry pick.

cc @linsun

istio-testing commented:
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/istio/istio/pull/14561#" title="Author self-approved">jmazzitelli</a>*
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **howardjohn**

If they are not already assigned, you can assign the PR to them by writing `/assign @howardjohn` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[install/OWNERS](https://github.com/istio/istio/blob/master/install/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>
<!-- META={"approvers":["howardjohn"]} -->

linsun commented:
@jmazzitelli thanks for the PR!   Quick q, i saw you removed user and passwordphase, how would user configure the id/pwd to logon to Kiali?

sdake commented on install/kubernetes/helm/istio/charts/kiali/templates/configmap.yaml:
```diff
@@ -11,12 +11,17 @@ metadata:
 data:
   config.yaml: |
     istio_namespace: {{ .Release.Namespace }}
+    auth:
```
are the auth strategies specified in some document somewhere?  IF so, can you add the link so we can document the change in the website and 1.2 if this is backported?

jmazzitelli replied:
This actually is backward compatible as-is - Kiali is going to behave the same as before (so for users installing via helm or upgrading, the behavior of Kiali doesn't change - the user will need to log into Kiali using credentials stored in the secret). The different auth strategies are documented here: https://www.kiali.io/documentation/getting-started/#_login_options - I can add that link to this configmap.yaml

Note: for Kubernetes users, the only ones applicable are "anonymous" and "login" (OpenShift has an additional one - "openshift" which utilizes the OpenShift OAuth client capability found only in OpenShift).

sdake commented on install/kubernetes/helm/istio/charts/kiali/values.yaml:
```diff
@@ -3,8 +3,8 @@
 replicaCount: 1
-hub: docker.io/kiali
-tag: v0.16
+hub: quay.io/kiali
```
can you offer some guidance on the permanent location for "new" images for kiali?  Clearly, we don't want to delete docker.io/kiali as it would damage our 1.1 releases in the field.  The question then becomes, will docker.io/kiali become a mirror, or will docker.io/kiali become deprecated (starting with this PR or has it already?)

As to using `quay.io/kiali` as the canonical source, that seems fine, but I'd like some clarity on how the images will be managed.  Will docker.io/kiali only contain old versions?

It is highly atypical that a hub would change anywhere across the cloud native ecosystem for an emerging or mature project and Istio is no exception to this rule.  Still, willing to work with the Kiali community if this is Kiali's desired path forward.

Can you please add clarity here?

jmazzitelli replied:
> Clearly, we don't want to delete docker.io/kiali as it would damage our 1.1 releases in the field.

The docker.io repository for Kiali is still there and will retain the old versions that are now currently. They will not be going away.

> will docker.io/kiali become a mirror, or will docker.io/kiali become deprecated (starting with this PR or has it already?)

Right now, docker.io is a mirror for us - we are now pushing Kiali images to BOTH docker.io and quay.io so as not to disrupt current users that may expect to still see Kiali images on docker.io - so you will still see Kiali images published to both. However, in the future, the plan is for Kiali to publish to quay.io as its canonical source. There are no plans right now for Kiali to stop publishing new images to docker.io, but we do want to start migrating our users over to using quay.io. And, again, old images will not be removed from docker.io regardless of this migration over to quay.io.

jmazzitelli commented:
> i saw you removed user and passwordphase, how would user configure the id/pwd to logon to Kiali?

@linsun those parameters are no longer used because we mount the secret differently now. In fact, I do not know why these got resurrected - I had deleted them in an earlier PR back in April - see https://github.com/istio/istio/pull/13312/files#diff-abe69812e356feaaefcdd5389b1be6d1L48 -- I guess that didn't get merged back into master.

The only places they were used were here: https://github.com/istio/istio/pull/13312/files#diff-94025c7c2b4beb94ee7f1a49ee588884L49 (and see a few lines down). Notice they were needed because we mounted the secret into env vars. Because we mount on the file system now, we don't need these. The keys are always assumed "username" and "passphrase".

Note also these aren't the actual username and passphrase - literally the values were "username" and "passphrase" since those were the keys in the secret that contains the values. But, as I explain above, we don't need the key values anymore in the deployment yaml, so these can be removed. 

To answer your question, "how would user configure the id/pwd to logon to Kiali?" they do it like before. Nothing changed in this respect. The user must create the secret with their own username and passphrase as we explain in the documentation here: https://istio.io/docs/tasks/telemetry/kiali/#create-a-secret

If you recall, we added a feature in the helm demo profile to allow helm to create a "demo secret" for the user - that has the credentials of "admin" / "admin". See this PR: https://github.com/istio/istio/pull/11715 So for users installing with the istio-demo yaml, they will get a secret "for free".

sdake commented:
> > i saw you removed user and passwordphase, how would user configure the id/pwd to logon to Kiali?
> 
> @linsun those parameters are no longer used because we mount the secret differently now. In fact, I do not know why these got resurrected - I had deleted them in an earlier PR back in April - see https://github.com/istio/istio/pull/13312/files#diff-abe69812e356feaaefcdd5389b1be6d1L48 -- I guess that didn't get merged back into master.
> 
> The only places they were used were here: https://github.com/istio/istio/pull/13312/files#diff-94025c7c2b4beb94ee7f1a49ee588884L49 (and see a few lines down). Notice they were needed because we mounted the secret into env vars. Because we mount on the file system now, we don't need these. The keys are always assumed "username" and "passphrase".
> 
> Note also these aren't the actual username and passphrase - literally the values were "username" and "passphrase" since those were the keys in the secret that contains the values. But, as I explain above, we don't need the key values anymore in the deployment yaml, so these can be removed.
> 
> To answer your question, "how would user configure the id/pwd to logon to Kiali?" they do it like before. Nothing changed in this respect. The user must create the secret with their own username and passphrase as we explain in the documentation here: https://istio.io/docs/tasks/telemetry/kiali/#create-a-secret
> 
> If you recall, we added a feature in the helm demo profile to allow helm to create a "demo secret" for the user - that has the credentials of "admin" / "admin". See this PR: #11715 So for users installing with the istio-demo yaml, they will get a secret "for free".

We need to watch out for PRs that pass credentials via env vars.  I am not convinced this model is secure "enough" - should always use secret refs.

istio-testing commented:
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/istio/istio/pull/14561#" title="Author self-approved">jmazzitelli</a>*, *<a href="https://github.com/istio/istio/pull/14561#pullrequestreview-245633632" title="Approved">sdake</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[install/OWNERS](https://github.com/istio/istio/blob/master/install/OWNERS)~~ [sdake]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>
<!-- META={"approvers":[]} -->

jmazzitelli commented:
the release 1.2 PR is found here https://github.com/istio/istio/pull/14567

istio-testing commented:
@jmazzitelli: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
prow/integ-galley-k8s-presubmit-tests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/integ-galley-k8s-presubmit-tests-master/1099) | `/test integ-galley-k8s-presubmit-tests-master`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@jmazzitelli: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
prow/integ-galley-k8s-presubmit-tests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/integ-galley-k8s-presubmit-tests-master/1099) | `/test integ-galley-k8s-presubmit-tests-master`
prow/istio-pilot-e2e-envoyv2-v1alpha3.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/istio-pilot-e2e-envoyv2-v1alpha3-master/1295) | `/test istio-pilot-e2e-envoyv2-v1alpha3-master`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@jmazzitelli: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
prow/integ-galley-k8s-presubmit-tests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/integ-galley-k8s-presubmit-tests-master/1099) | `/test integ-galley-k8s-presubmit-tests-master`
prow/istio-pilot-e2e-envoyv2-v1alpha3.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/istio-pilot-e2e-envoyv2-v1alpha3-master/1295) | `/test istio-pilot-e2e-envoyv2-v1alpha3-master`
prow/e2e-bookInfoTests-trustdomain.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/e2e-bookInfoTests-trustdomain-master/1099) | `/test e2e-bookInfoTests-trustdomain-master`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@jmazzitelli: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
prow/integ-galley-k8s-presubmit-tests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/integ-galley-k8s-presubmit-tests-master/1099) | `/test integ-galley-k8s-presubmit-tests-master`
prow/istio-pilot-e2e-envoyv2-v1alpha3.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/istio-pilot-e2e-envoyv2-v1alpha3-master/1295) | `/test istio-pilot-e2e-envoyv2-v1alpha3-master`
prow/e2e-bookInfoTests-trustdomain.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/e2e-bookInfoTests-trustdomain-master/1099) | `/test e2e-bookInfoTests-trustdomain-master`
prow/e2e-simpleTests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/e2e-simpleTests-master/1371) | `/test e2e-simpleTests-master`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@jmazzitelli: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
prow/integ-galley-k8s-presubmit-tests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/integ-galley-k8s-presubmit-tests-master/1099) | `/test integ-galley-k8s-presubmit-tests-master`
prow/istio-pilot-e2e-envoyv2-v1alpha3.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/istio-pilot-e2e-envoyv2-v1alpha3-master/1295) | `/test istio-pilot-e2e-envoyv2-v1alpha3-master`
prow/e2e-bookInfoTests-trustdomain.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/e2e-bookInfoTests-trustdomain-master/1099) | `/test e2e-bookInfoTests-trustdomain-master`
prow/e2e-simpleTests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/e2e-simpleTests-master/1371) | `/test e2e-simpleTests-master`
prow/e2e-dashboard.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/e2e-dashboard-master/1285) | `/test e2e-dashboard-master`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@jmazzitelli: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
prow/integ-galley-k8s-presubmit-tests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/integ-galley-k8s-presubmit-tests-master/1099) | `/test integ-galley-k8s-presubmit-tests-master`
prow/istio-pilot-e2e-envoyv2-v1alpha3.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/istio-pilot-e2e-envoyv2-v1alpha3-master/1295) | `/test istio-pilot-e2e-envoyv2-v1alpha3-master`
prow/e2e-bookInfoTests-trustdomain.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/e2e-bookInfoTests-trustdomain-master/1099) | `/test e2e-bookInfoTests-trustdomain-master`
prow/e2e-simpleTests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/e2e-simpleTests-master/1371) | `/test e2e-simpleTests-master`
prow/e2e-dashboard.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/e2e-dashboard-master/1285) | `/test e2e-dashboard-master`
prow/e2e-simpleTests-minProfile.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/e2e-simpleTestsMinProfile-master/1114) | `/test e2e-simpleTestsMinProfile-master`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@jmazzitelli: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
prow/integ-galley-k8s-presubmit-tests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/integ-galley-k8s-presubmit-tests-master/1099) | `/test integ-galley-k8s-presubmit-tests-master`
prow/istio-pilot-e2e-envoyv2-v1alpha3.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/istio-pilot-e2e-envoyv2-v1alpha3-master/1295) | `/test istio-pilot-e2e-envoyv2-v1alpha3-master`
prow/e2e-bookInfoTests-trustdomain.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/e2e-bookInfoTests-trustdomain-master/1099) | `/test e2e-bookInfoTests-trustdomain-master`
prow/e2e-simpleTests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/e2e-simpleTests-master/1371) | `/test e2e-simpleTests-master`
prow/e2e-dashboard.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/e2e-dashboard-master/1285) | `/test e2e-dashboard-master`
prow/e2e-simpleTests-minProfile.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/e2e-simpleTestsMinProfile-master/1114) | `/test e2e-simpleTestsMinProfile-master`
prow/e2e-bookInfoTests-v1alpha3.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/e2e-bookInfoTests-envoyv2-v1alpha3-master/1284) | `/test e2e-bookInfoTests-envoyv2-v1alpha3-master`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@jmazzitelli: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
prow/integ-galley-k8s-presubmit-tests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/integ-galley-k8s-presubmit-tests-master/1099) | `/test integ-galley-k8s-presubmit-tests-master`
prow/istio-pilot-e2e-envoyv2-v1alpha3.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/istio-pilot-e2e-envoyv2-v1alpha3-master/1295) | `/test istio-pilot-e2e-envoyv2-v1alpha3-master`
prow/e2e-bookInfoTests-trustdomain.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/e2e-bookInfoTests-trustdomain-master/1099) | `/test e2e-bookInfoTests-trustdomain-master`
prow/e2e-simpleTests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/e2e-simpleTests-master/1371) | `/test e2e-simpleTests-master`
prow/e2e-dashboard.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/e2e-dashboard-master/1285) | `/test e2e-dashboard-master`
prow/e2e-simpleTests-minProfile.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/e2e-simpleTestsMinProfile-master/1114) | `/test e2e-simpleTestsMinProfile-master`
prow/e2e-bookInfoTests-v1alpha3.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/e2e-bookInfoTests-envoyv2-v1alpha3-master/1284) | `/test e2e-bookInfoTests-envoyv2-v1alpha3-master`
prow/istio-pilot-multicluster-e2e.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/istio-pilot-multicluster-e2e-master/1093) | `/test istio-pilot-multicluster-e2e-master`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@jmazzitelli: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
prow/integ-galley-k8s-presubmit-tests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/integ-galley-k8s-presubmit-tests-master/1099) | `/test integ-galley-k8s-presubmit-tests-master`
prow/istio-pilot-e2e-envoyv2-v1alpha3.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/istio-pilot-e2e-envoyv2-v1alpha3-master/1295) | `/test istio-pilot-e2e-envoyv2-v1alpha3-master`
prow/e2e-bookInfoTests-trustdomain.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/e2e-bookInfoTests-trustdomain-master/1099) | `/test e2e-bookInfoTests-trustdomain-master`
prow/e2e-simpleTests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/e2e-simpleTests-master/1371) | `/test e2e-simpleTests-master`
prow/e2e-dashboard.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/e2e-dashboard-master/1285) | `/test e2e-dashboard-master`
prow/e2e-simpleTests-minProfile.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/e2e-simpleTestsMinProfile-master/1114) | `/test e2e-simpleTestsMinProfile-master`
prow/e2e-bookInfoTests-v1alpha3.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/e2e-bookInfoTests-envoyv2-v1alpha3-master/1284) | `/test e2e-bookInfoTests-envoyv2-v1alpha3-master`
prow/istio-pilot-multicluster-e2e.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/istio-pilot-multicluster-e2e-master/1093) | `/test istio-pilot-multicluster-e2e-master`
prow/integ-pilot-k8s-presubmit-tests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/integ-pilot-k8s-presubmit-tests-master/1134) | `/test integ-pilot-k8s-presubmit-tests-master`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@jmazzitelli: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
prow/integ-galley-k8s-presubmit-tests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/integ-galley-k8s-presubmit-tests-master/1099) | `/test integ-galley-k8s-presubmit-tests-master`
prow/istio-pilot-e2e-envoyv2-v1alpha3.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/istio-pilot-e2e-envoyv2-v1alpha3-master/1295) | `/test istio-pilot-e2e-envoyv2-v1alpha3-master`
prow/e2e-bookInfoTests-trustdomain.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/e2e-bookInfoTests-trustdomain-master/1099) | `/test e2e-bookInfoTests-trustdomain-master`
prow/e2e-simpleTests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/e2e-simpleTests-master/1371) | `/test e2e-simpleTests-master`
prow/e2e-dashboard.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/e2e-dashboard-master/1285) | `/test e2e-dashboard-master`
prow/e2e-simpleTests-minProfile.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/e2e-simpleTestsMinProfile-master/1114) | `/test e2e-simpleTestsMinProfile-master`
prow/e2e-bookInfoTests-v1alpha3.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/e2e-bookInfoTests-envoyv2-v1alpha3-master/1284) | `/test e2e-bookInfoTests-envoyv2-v1alpha3-master`
prow/istio-pilot-multicluster-e2e.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/istio-pilot-multicluster-e2e-master/1093) | `/test istio-pilot-multicluster-e2e-master`
prow/integ-pilot-k8s-presubmit-tests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/integ-pilot-k8s-presubmit-tests-master/1134) | `/test integ-pilot-k8s-presubmit-tests-master`
prow/integ-security-k8s-presubmit-tests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/integ-security-k8s-presubmit-tests-master/1104) | `/test integ-security-k8s-presubmit-tests-master`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

fpesce commented:
/retest

sdake commented:
/retest

istio-testing commented:
@linsun: changing LGTM is restricted to assignees, and assigning you to the PR failed.

<details>

In response to [this](https://github.com/istio/istio/pull/14561#pullrequestreview-245779618):

>/lgtm
>/approve
>
>thank you @jmazzitelli for your comment-- now we just need to get test passing!


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

istio-testing commented:
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/istio/istio/pull/14561#" title="Author self-approved">jmazzitelli</a>*, *<a href="https://github.com/istio/istio/pull/14561#pullrequestreview-245779618" title="Approved">linsun</a>*, *<a href="https://github.com/istio/istio/pull/14561#pullrequestreview-245633632" title="Approved">sdake</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[install/OWNERS](https://github.com/istio/istio/blob/master/install/OWNERS)~~ [linsun,sdake]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>
<!-- META={"approvers":[]} -->

jmazzitelli commented:
FYI: I cherry-picked this over to the release-1.2 branch on my local machine and ran it in the latest minikube and it worked (here's the Kiali About box: you can see "unknown Istio" version is simply the Istio on my local machine - this is the head of release-1.2 branch, not a particular release):

![image](https://user-images.githubusercontent.com/2029470/58927756-dd69a380-871d-11e9-88bc-c08b09cd2c82.png)

howardjohn commented:
/test all

istio-testing commented:
@jmazzitelli: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
prow/integ-security-k8s-presubmit-tests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/integ-security-k8s-presubmit-tests-master/1104) | `/test integ-security-k8s-presubmit-tests-master`
prow/integ-galley-k8s-presubmit-tests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/integ-galley-k8s-presubmit-tests-master/1199) | `/test integ-galley-k8s-presubmit-tests-master`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@jmazzitelli: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
prow/integ-security-k8s-presubmit-tests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/integ-security-k8s-presubmit-tests-master/1104) | `/test integ-security-k8s-presubmit-tests-master`
prow/integ-galley-k8s-presubmit-tests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/integ-galley-k8s-presubmit-tests-master/1199) | `/test integ-galley-k8s-presubmit-tests-master`
prow/integ-istioctl-k8s-presubmit-tests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/integ-istioctl-k8s-presubmit-tests-master/183) | `/test integ-istioctl-k8s-presubmit-tests-master`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@jmazzitelli: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
prow/integ-security-k8s-presubmit-tests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/integ-security-k8s-presubmit-tests-master/1104) | `/test integ-security-k8s-presubmit-tests-master`
prow/integ-galley-k8s-presubmit-tests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/integ-galley-k8s-presubmit-tests-master/1199) | `/test integ-galley-k8s-presubmit-tests-master`
prow/integ-istioctl-k8s-presubmit-tests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/integ-istioctl-k8s-presubmit-tests-master/183) | `/test integ-istioctl-k8s-presubmit-tests-master`
prow/e2e-dashboard.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/e2e-dashboard-master/1402) | `/test e2e-dashboard-master`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@jmazzitelli: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
prow/integ-security-k8s-presubmit-tests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/integ-security-k8s-presubmit-tests-master/1104) | `/test integ-security-k8s-presubmit-tests-master`
prow/integ-galley-k8s-presubmit-tests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/integ-galley-k8s-presubmit-tests-master/1199) | `/test integ-galley-k8s-presubmit-tests-master`
prow/integ-istioctl-k8s-presubmit-tests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/integ-istioctl-k8s-presubmit-tests-master/183) | `/test integ-istioctl-k8s-presubmit-tests-master`
prow/e2e-dashboard.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/e2e-dashboard-master/1402) | `/test e2e-dashboard-master`
prow/integ-pilot-k8s-presubmit-tests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/integ-pilot-k8s-presubmit-tests-master/1235) | `/test integ-pilot-k8s-presubmit-tests-master`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@jmazzitelli: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
prow/integ-security-k8s-presubmit-tests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/integ-security-k8s-presubmit-tests-master/1104) | `/test integ-security-k8s-presubmit-tests-master`
prow/integ-galley-k8s-presubmit-tests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/integ-galley-k8s-presubmit-tests-master/1199) | `/test integ-galley-k8s-presubmit-tests-master`
prow/integ-istioctl-k8s-presubmit-tests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/integ-istioctl-k8s-presubmit-tests-master/183) | `/test integ-istioctl-k8s-presubmit-tests-master`
prow/e2e-dashboard.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/e2e-dashboard-master/1402) | `/test e2e-dashboard-master`
prow/integ-pilot-k8s-presubmit-tests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/integ-pilot-k8s-presubmit-tests-master/1235) | `/test integ-pilot-k8s-presubmit-tests-master`
prow/e2e-mixer-no_auth.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/e2e-mixer-no_auth-master/1419) | `/test e2e-mixer-no_auth-master`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@jmazzitelli: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
prow/integ-security-k8s-presubmit-tests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/integ-security-k8s-presubmit-tests-master/1104) | `/test integ-security-k8s-presubmit-tests-master`
prow/integ-galley-k8s-presubmit-tests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/integ-galley-k8s-presubmit-tests-master/1199) | `/test integ-galley-k8s-presubmit-tests-master`
prow/integ-istioctl-k8s-presubmit-tests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/integ-istioctl-k8s-presubmit-tests-master/183) | `/test integ-istioctl-k8s-presubmit-tests-master`
prow/e2e-dashboard.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/e2e-dashboard-master/1402) | `/test e2e-dashboard-master`
prow/integ-pilot-k8s-presubmit-tests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/integ-pilot-k8s-presubmit-tests-master/1235) | `/test integ-pilot-k8s-presubmit-tests-master`
prow/e2e-mixer-no_auth.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/e2e-mixer-no_auth-master/1419) | `/test e2e-mixer-no_auth-master`
prow/istio-pilot-e2e-envoyv2-v1alpha3.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/istio-pilot-e2e-envoyv2-v1alpha3-master/1409) | `/test istio-pilot-e2e-envoyv2-v1alpha3-master`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@jmazzitelli: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
prow/integ-security-k8s-presubmit-tests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/integ-security-k8s-presubmit-tests-master/1104) | `/test integ-security-k8s-presubmit-tests-master`
prow/integ-galley-k8s-presubmit-tests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/integ-galley-k8s-presubmit-tests-master/1199) | `/test integ-galley-k8s-presubmit-tests-master`
prow/integ-istioctl-k8s-presubmit-tests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/integ-istioctl-k8s-presubmit-tests-master/183) | `/test integ-istioctl-k8s-presubmit-tests-master`
prow/e2e-dashboard.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/e2e-dashboard-master/1402) | `/test e2e-dashboard-master`
prow/integ-pilot-k8s-presubmit-tests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/integ-pilot-k8s-presubmit-tests-master/1235) | `/test integ-pilot-k8s-presubmit-tests-master`
prow/e2e-mixer-no_auth.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/e2e-mixer-no_auth-master/1419) | `/test e2e-mixer-no_auth-master`
prow/istio-pilot-e2e-envoyv2-v1alpha3.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/istio-pilot-e2e-envoyv2-v1alpha3-master/1409) | `/test istio-pilot-e2e-envoyv2-v1alpha3-master`
prow/e2e-bookInfoTests-v1alpha3.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/e2e-bookInfoTests-envoyv2-v1alpha3-master/1400) | `/test e2e-bookInfoTests-envoyv2-v1alpha3-master`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@jmazzitelli: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
prow/integ-security-k8s-presubmit-tests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/integ-security-k8s-presubmit-tests-master/1104) | `/test integ-security-k8s-presubmit-tests-master`
prow/integ-galley-k8s-presubmit-tests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/integ-galley-k8s-presubmit-tests-master/1199) | `/test integ-galley-k8s-presubmit-tests-master`
prow/integ-istioctl-k8s-presubmit-tests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/integ-istioctl-k8s-presubmit-tests-master/183) | `/test integ-istioctl-k8s-presubmit-tests-master`
prow/e2e-dashboard.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/e2e-dashboard-master/1402) | `/test e2e-dashboard-master`
prow/integ-pilot-k8s-presubmit-tests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/integ-pilot-k8s-presubmit-tests-master/1235) | `/test integ-pilot-k8s-presubmit-tests-master`
prow/e2e-mixer-no_auth.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/e2e-mixer-no_auth-master/1419) | `/test e2e-mixer-no_auth-master`
prow/istio-pilot-e2e-envoyv2-v1alpha3.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/istio-pilot-e2e-envoyv2-v1alpha3-master/1409) | `/test istio-pilot-e2e-envoyv2-v1alpha3-master`
prow/e2e-bookInfoTests-v1alpha3.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/e2e-bookInfoTests-envoyv2-v1alpha3-master/1400) | `/test e2e-bookInfoTests-envoyv2-v1alpha3-master`
prow/e2e_pilotv2_auth_sds.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/istio_auth_sds_e2e-master/1399) | `/test istio_auth_sds_e2e-master`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@jmazzitelli: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
prow/integ-security-k8s-presubmit-tests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/integ-security-k8s-presubmit-tests-master/1104) | `/test integ-security-k8s-presubmit-tests-master`
prow/integ-galley-k8s-presubmit-tests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/integ-galley-k8s-presubmit-tests-master/1199) | `/test integ-galley-k8s-presubmit-tests-master`
prow/integ-istioctl-k8s-presubmit-tests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/integ-istioctl-k8s-presubmit-tests-master/183) | `/test integ-istioctl-k8s-presubmit-tests-master`
prow/e2e-dashboard.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/e2e-dashboard-master/1402) | `/test e2e-dashboard-master`
prow/integ-pilot-k8s-presubmit-tests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/integ-pilot-k8s-presubmit-tests-master/1235) | `/test integ-pilot-k8s-presubmit-tests-master`
prow/e2e-mixer-no_auth.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/e2e-mixer-no_auth-master/1419) | `/test e2e-mixer-no_auth-master`
prow/istio-pilot-e2e-envoyv2-v1alpha3.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/istio-pilot-e2e-envoyv2-v1alpha3-master/1409) | `/test istio-pilot-e2e-envoyv2-v1alpha3-master`
prow/e2e-bookInfoTests-v1alpha3.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/e2e-bookInfoTests-envoyv2-v1alpha3-master/1400) | `/test e2e-bookInfoTests-envoyv2-v1alpha3-master`
prow/e2e_pilotv2_auth_sds.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/istio_auth_sds_e2e-master/1399) | `/test istio_auth_sds_e2e-master`
prow/istio-pilot-multicluster-e2e.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/istio-pilot-multicluster-e2e-master/1197) | `/test istio-pilot-multicluster-e2e-master`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@jmazzitelli: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
prow/integ-galley-k8s-presubmit-tests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/integ-galley-k8s-presubmit-tests-master/1199) | `/test integ-galley-k8s-presubmit-tests-master`
prow/integ-istioctl-k8s-presubmit-tests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/integ-istioctl-k8s-presubmit-tests-master/183) | `/test integ-istioctl-k8s-presubmit-tests-master`
prow/e2e-dashboard.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/e2e-dashboard-master/1402) | `/test e2e-dashboard-master`
prow/integ-pilot-k8s-presubmit-tests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/integ-pilot-k8s-presubmit-tests-master/1235) | `/test integ-pilot-k8s-presubmit-tests-master`
prow/e2e-mixer-no_auth.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/e2e-mixer-no_auth-master/1419) | `/test e2e-mixer-no_auth-master`
prow/istio-pilot-e2e-envoyv2-v1alpha3.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/istio-pilot-e2e-envoyv2-v1alpha3-master/1409) | `/test istio-pilot-e2e-envoyv2-v1alpha3-master`
prow/e2e-bookInfoTests-v1alpha3.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/e2e-bookInfoTests-envoyv2-v1alpha3-master/1400) | `/test e2e-bookInfoTests-envoyv2-v1alpha3-master`
prow/e2e_pilotv2_auth_sds.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/istio_auth_sds_e2e-master/1399) | `/test istio_auth_sds_e2e-master`
prow/istio-pilot-multicluster-e2e.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/istio-pilot-multicluster-e2e-master/1197) | `/test istio-pilot-multicluster-e2e-master`
prow/integ-security-k8s-presubmit-tests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/integ-security-k8s-presubmit-tests-master/1212) | `/test integ-security-k8s-presubmit-tests-master`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

howardjohn commented:
/test all

istio-testing commented:
@jmazzitelli: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
prow/e2e-mixer-no_auth.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/e2e-mixer-no_auth-master/1419) | `/test e2e-mixer-no_auth-master`
prow/e2e-bookInfoTests-v1alpha3.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/e2e-bookInfoTests-envoyv2-v1alpha3-master/1400) | `/test e2e-bookInfoTests-envoyv2-v1alpha3-master`
prow/integ-security-k8s-presubmit-tests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/integ-security-k8s-presubmit-tests-master/1212) | `/test integ-security-k8s-presubmit-tests-master`
prow/integ-galley-k8s-presubmit-tests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/integ-galley-k8s-presubmit-tests-master/1236) | `/test integ-galley-k8s-presubmit-tests-master`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@jmazzitelli: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
prow/integ-galley-k8s-presubmit-tests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/integ-galley-k8s-presubmit-tests-master/1236) | `/test integ-galley-k8s-presubmit-tests-master`
prow/integ-security-k8s-presubmit-tests.sh | 85a6e6a79beae252b11a61674d46804b554d9ecd | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/integ-security-k8s-presubmit-tests-master/1238) | `/test integ-security-k8s-presubmit-tests-master`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

jmazzitelli commented:
The [last test failure](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/14561/integ-galley-k8s-presubmit-tests-master/1236) is: `fatal: unable to access 'https://github.com/istio/istio.git/': Could not resolve host: github.com` </me scratches head>

jmazzitelli commented:
/test integ-galley-k8s-presubmit-tests-master

jmazzitelli commented:
It said "job aborted" now? Trying again:

/test integ-galley-k8s-presubmit-tests-master

ericvn commented:
Submitted the test again since the earlier one was going to abort, so short-circuited that and resubmitting.
